examples/index.html

```diff
@@ -46,6 +46,7 @@
       </auto-complete>
       <button type="submit">Save</button>
     </form>
-    <script src="../dist/index.umd.js"></script>
+    <!-- <script src="../dist/index.umd.js"></script> -->
+    <script src="https://unpkg.com/@github/auto-complete-element@0.7.0/dist/index.umd.js"></script>
```

Comment:
It should use the latest version so we don't have to bump it every time. 

```suggestion
    <script src="https://unpkg.com/@github/auto-complete-element@latest/dist/index.umd.js"></script>
```